### DISCUSSION TITLE
differentiate fire spread and fire extinguishing

### DISCUSSION THREAD

**Discussion by emielderckx** [PurpurMC/Purpur#1228] [Feature Requests]
When doFireTick is set to false, differentiate fire spread and extinguishing.

Would like fire spread to be turned off, but still want fire to normally extinguish. So some options to specify this would be great.

**Comment by SmallSansSerif** [comment]
Something like this would be great added into Purpur. 

https://modrinth.com/mod/fire-spread-tweaks
